openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/TagsTask.tsx

```diff
@@ -69,60 +70,57 @@ const TagsTask: FC<TagsTaskProps> = ({
         />
       );
     }
-  };
+  }, [oldValue, newValue]);
 
   /**
    *
    * @returns Suggested tags diff
    */
-  const getSuggestedTagDiff = () => {
-    const newTags = task?.suggestion;
-    const oldTags = task?.oldValue;
-
-    return !newTags && !oldTags ? (
-      <span className="tw-p-2 tw-text-grey-muted">
-        {t('label.no-entity', { entity: t('label.suggestion') })}
-      </span>
-    ) : (
-      <TagsDiffView
-        diffArr={diffArrays(
-          JSON.parse(oldTags ?? '[]'),
-          JSON.parse(newTags ?? '[]')
-        )}
-      />
-    );
-  };
+  const getSuggestedTagDiff = useMemo(() => {
```

Comment:
```suggestion
  const suggestedTagsDiff = useMemo(() => {
```
It's property not method